### PR TITLE
Adding more disables.

### PR DESCRIPTION
**Pull request by datamattsson**

### CODE REVIEW

File: helm/charts/hpe-csi-driver/templates/hpe-csi-node.yaml [hunk 1]
```diff
@@ -130,6 +142,10 @@ spec:
             - name: DISABLE_NODE_CONFIGURATION
               value: "true"
             {{- end }}
+            {{ if .Values.disableNodeMonitor -}}
+            - name: DISABLE_NODE_MONITOR
+              value: "true"
```
Comment (AnushaY1916) [inline helm/charts/hpe-csi-driver/templates/hpe-csi-node.yaml:147]: The value is set false everywhere else

Comment (datamattsson) [reply helm/charts/hpe-csi-driver/templates/hpe-csi-node.yaml:147]: The default behavior is to have the Node Monitor enabled, hence false. If a user elects to `disableNodeMonitor`, it's set to true.

**Comment by AnushaY1916** [comment]
DISABLE_NODE_MONITOR should be set in edge yaml files as well (both init container as well as hpe-csi-driver)

**Review by AnushaY1916 — APPROVED** [review]
LGTM

**Pull request merged**